File: backend-maven/src/main/java/com/mu54omd/mini_ecommerce/backend_maven/dto/OrderResponse.java
package com.mu54omd.mini_ecommerce.backend_maven.dto;

import java.time.LocalDateTime;
import java.util.List;

public record OrderResponse(
        Long id,
        String username,
        LocalDateTime createdAt,
        String status,
        double totalPrice,
        List<OrderItemResponse> items
) {}

File: frontend-gradle/composeApp/src/commonMain/kotlin/com/mu54omd/mini_ecommerce/frontend_gradle/data/models/OrderResponse.kt
package com.mu54omd.mini_ecommerce.frontend_gradle.data.models

import kotlinx.serialization.Serializable

@Serializable
data class OrderResponse(
    val id: Long,
    val username: String,
    val items: List<OrderItemResponse>,
    val totalPrice: Double,
    val status: String,
)